packages/frontend/src/components/DashboardTiles/EditChartMenuItem.tsx

```diff
@@ -28,11 +28,19 @@ const EditChartMenuItem: FC<Props> = ({ tile, ...props }) => {
     }>();
 
     const userCanManageExplore = user.data?.ability?.can('manage', 'Explore');
+    const userCanManageSavedChart = user.data?.ability?.can(
+        'manage',
+        'SavedChart',
+    );
 
-    if (!tile.properties.savedChartUuid || !userCanManageExplore) return null;
+    const cannotEditChart =
+        !tile.properties.savedChartUuid ||
+        !userCanManageExplore ||
+        !userCanManageSavedChart;
 
     return (
         <LinkMenuItem
+            disabled={cannotEditChart}
```

Comment:
We usually hide options the user doesn't have permission to instead of disabling them.

Reply:
Cool! makes sense as to why this wasn't there before (disabling an option)

Reply:
reverted: https://github.com/lightdash/lightdash/pull/8519/commits/15de57f61b2b16232d83f6388e7933867981834d